src/utils/parse-fixture.js

```diff
@@ -0,0 +1,142 @@
+import { parse } from 'babylon';
+import { GeneralError } from '../errors/runtime';
+
+import promisify from '../utils/promisify';
+import fs from 'fs';
+
+import MESSAGE from '../errors/runtime/message';
+
+var readFile = promisify(fs.readFile);
+
+const FIXTURE_RE = /(^|;|\s+)fixture\s*(\.|\(|`)/;
+const TEST_RE    = /(^|;|\s+)test\s*(\.|\()/;
```

Comment:
It's already declared in `compiler/es-next`, also this REs are out of date (they don't work with modifiers like `skip`)